types/idMacros.d.ts

```diff
@@ -4,4 +4,4 @@
 /**
  * This variable will be automatically replaced by the "version" string from the package.json file
  */
-declare const PKG_VERSION: string;
+declare const $PKG_VERSION: string;
```

Comment:
Looking at this more, I don't think this macro fits in roblox-ts. It's more something for `rbxts-transform-debug` or the like. See the same reasoning for removal at `opcall`.